### DISCUSSION TITLE
use tasmota as the zigbee gateway，how to get zigbee sub-devices status.

### DISCUSSION THREAD

**Discussion by JaredLees** [arendst/Tasmota#17063] [Q&A]
Hello, I use tasmota as the zigbee gateway, the gateway is connected to mqtt server (EMQX), the gateway is connected to three zigbee devices, I want to get the status of three zigbee sub-devices (online or offline), may I ask how to subscribe to get the status of the sub-devices（online or offline）。

**Comment by s-hadinger** [comment]
Unfortunately that's not how zigbee works. Battery powered devices report only once in a while to save battery. You can't tell whether they're online or offline. Only when is the last time they connected
